localization/english/event_localization/wc_draenor_events_l_english.yml

```diff
@@ -23,3 +23,13 @@
  WCDRA_25_TITLE:0 "The Dark Portal Reopened"
  WCDRA_25_DESC:0 "WRITE ME"
  WCDRA_25_OPT_A:0 "WRITE ME"
+
+ WCDRA_200_TITLE:0 "Intruders Beyond the [ROOT.Char.Custom('GetPortalRift')]"
+ WCDRA_200_DESC_beginning:0 "Your soldiers report they faced armed forces coming out of the [ROOT.Char.Custom('GetDarkPortalRift')]. So far their numbers are unknown, but it's clear there is some movement beyond the veil.\n\nRumors say these aliens are"
+ WCDRA_200_DESC_draenei:0 "draenei survivors led by a powerful exarch blessed by the naaru."
+ WCDRA_200_DESC_arakkoa:0 "arakkoa outcasts led by a reincarnation of Terokk, the greatest king."
+ WCDRA_200_DESC_orcs:0 "members of wild and despair orcish clans gathered together."
+ WCDRA_200_DESC_ogres:0 "members of wild and despair ogre clans gathered together."
+ WCDRA_200_DESC_ending:0 "They call to storm the [ROOT.Char.Custom('GetDarkPortalRift')] and get a better place in the new world.\n\nImprisoned intruders tell they have no intention to fight for the [ROOT.Char.Custom('GetPortalRift')]. Draenor is on the edge of peril. They want nothing more than fertile land and are ready to seize it in combat."
+ WCDRA_200_OPT_A:0 "Settle down and let them through peacefully."
+ WCDRA_200_OPT_B:0 "Only over my dead body."
```

Comment:
Change the wording:
"Over my dead body."

Reply:
```suggestion
 WCDRA_200_OPT_B:0 "Over my dead body."
```